sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/models/EncryptionContainerResponse.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.EncryptionCosmosAsyncContainer;
+
+public class EncryptionContainerResponse {
```

Comment:
why can't we use existing `CosmosContainerResponse`
why do we need another public type?